lib/wallet/open_channel.dart

```diff
@@ -44,57 +45,105 @@ class _OpenChannelState extends State<OpenChannel> {
-              const Text("Node Name", style: TextStyle(color: Colors.grey)),
-              const SizedBox(
-                height: 5.0,
-              ),
-              const SelectableText('10101', style: TextStyle(fontSize: 20)),
-              const SizedBox(
-                height: 10.0,
-              ),
-              const Text("Your Bitcoin Balance", style: TextStyle(color: Colors.grey)),
-              const SizedBox(
-                height: 5.0,
-              ),
-              SelectableText(bitcoinBalance.amount.asSats.toString(),
-                  style: const TextStyle(fontSize: 20)),
-              const SizedBox(
-                height: 10.0,
-              ),
-              const Text("Channel Amount", style: TextStyle(color: Colors.grey)),
-              // TODO: Form validation
-              TextFormField(
-                initialValue: takerChannelAmount.toString(),
-                keyboardType: TextInputType.number,
-                decoration: const InputDecoration(
-                  border: UnderlineInputBorder(),
-                ),
-                inputFormatters: <TextInputFormatter>[FilteringTextInputFormatter.digitsOnly],
-                onChanged: (text) {
-                  setState(() {
-                    takerChannelAmount =
-                        text != "" ? int.parse(text).clamp(0, OpenChannel.maxChannelAmount) : 0;
-                  });
-                },
+            Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: const [
+              Balance(balanceSelector: BalanceSelector.bitcoin),
+              SizedBox(height: 10),
+              Padding(
+                padding: EdgeInsets.only(left: 20.0, right: 20.0),
+                child: Divider(),
               )
             ]),
+            const SizedBox(height: 10),
+            const Center(child: Icon(Icons.private_connectivity_outlined, size: 80)),
+            // const Center(child: Text("How this works", style: TextStyle(fontWeight: FontWeight.w600, fontSize: 24))),
+            const SizedBox(height: 10),
+            // const Center(child: Text("Before trading non-custodial, we need to establish a secure lightning channel.", style: TextStyle(fontWeight: FontWeight.normal, fontSize: 16))),
+            // const SizedBox(height: 15),
+            ListTile(
+                leading: Container(
+                  width: 50,
+                  decoration:
+                      BoxDecoration(color: Colors.blue, borderRadius: BorderRadius.circular(100)),
+                  child: const Center(
+                      child: Text(
+                    "1",
+                    style: TextStyle(color: Colors.white, fontSize: 20),
+                  )),
+                ),
+                title: const Text("Fund your bitcoin wallet"),
+                subtitle: const Text(
+                    "In order to establish a secure channel between 10101 and you we need to lock some Bitcoin on chain through a Lightning channel.",
```

Comment:
```suggestion
                    "In order to establish a Lightning channel between 10101 and you we need to lock some Bitcoin on chain.",
```

Reply:
Bitcoin should be lowercase when it refers to the coins themselves imo.

Reply:
I like it uppercase, but that's probably the German in me 🤪